app/src/focus/res/drawable-xhdpi/background_gradient_dark.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why is this in `drawable-xhdpi`?

Reply:
oh, weird. It looks like it's in the base `drawable` folder on android studio